src/main/java/FireboltSqlDialect.java

```diff
@@ -36,11 +35,15 @@
 import java.util.List;
 import java.util.Locale;
 import java.util.regex.Pattern;
+
+import static org.apache.calcite.sql.fun.SqlBetweenOperator.*;
+
 /**
  * A SqlDialect implementation for the Firebolt database.
  */
 public class FireboltSqlDialect extends SqlDialect {
-    public static final SqlDialect.Context DEFAULT_CONTEXT = SqlDialect.EMPTY_CONTEXT
+    public static final Context DEFAULT_CONTEXT = SqlDialect.EMPTY_CONTEXT
+            //.withDatabaseProduct("FIREBOLT") //DatabaseProduct.FIREBOLT
```

Comment:
comment

Reply:
This comment is expected as it is the diff with what we have in calcite (to make it easier to keep the files in sync) but needs to be commented out so that the file can be compiled